bash-debugger/bdb.sh

```diff
@@ -1,36 +1,75 @@
 #!/bin/bash
+# shellcheck disable=SC2162,SC1090
+
+_e() { echo $'\e['"${1:-}"m ; }
```

Comment:
I'm a little bit hesitant to add color support since this may conflict with the logic of the script we debug script that also uses color output. Maybe color output can be optional?

Reply:
fair enough. what do you think is the best way to get implement it? environment variable (because we don't have a config) or an argument?

Reply:
I think environment variable will work better for this case. We can have something like `BDB_COLOR_OUTPUT` and if it is not empty - do output in color. What do you think?

Reply:
done :)